### ISSUE TITLE
Formating github team auth config flag for fly set-team

### ISSUE THREAD

**Issue by xtremerui** [concourse/concourse#3453]
There is a case that `fly set-team --github-team github:some-org:some-team` could actually work and cause a mis-config in team auth.

It could help if fly could verify the format of `--github-team` is at least `xxxxx:xxxxx`. Similar to fly execute [--input-from](https://github.com/concourse/concourse/blob/3a4d437da38ff0d930c3bdf36ff0dfbdca6c0762/fly/commands/internal/flaghelpers/job_flag.go#L19-L38)

**Comment by vito** [comment]
This is a bit tricky as GitHub teams can technically have `:` in their names.

**Comment by xtremerui** [comment]
@vito seems like when creating Github team if you put `:` in team name, its handle is actually replacing `:` with `-`

![Screen Shot 2019-03-11 at 9 38 34 AM](https://user-images.githubusercontent.com/1585949/54128108-b7b05500-43e1-11e9-94db-b41da265096a.png)

Guess it should be ok for concourse to use `:` as delimiter?

**Comment by vito** [comment]
@xtremerui True. We allow both team names *and* slugs, so I guess you could just specify the slug instead if you've got a `:` in the name.

**Comment by xtremerui** [comment]
Pause this due to https://github.com/concourse/concourse/pull/3702#discussion_r273791190

**Comment by taylorsilva** [comment]
# Issue Bankruptcy

This issue is being closed as part of 2025's **Issue Bankruptcy** declaration. [See here for details](https://blog.concourse-ci.org/posts/2025-01-06-concourse-2024-review/)

If you think this issue still has merit then feel free to open a new issue and reference this one.
